### PR TITLE
Adding workload type to name and slack integration

### PR DESCRIPTION
**Pull request by paigerube14**
Adding slack integration into job itself as the old way continues to get outdated

Also with Nathans help I added the workload name to the displayed name of the job

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -246,6 +251,7 @@ pipeline {
                 buildinfo.params.each { env.setProperty(it.key, it.value) }
             }
             script {
+                currentBuild.displayName = "${currentBuild.displayName}-${params.WORKLOAD}"
```
Comment (nathan-weinberg) [inline Jenkinsfile:254]: Should we just have this be a one-liner in L249?

File: Jenkinsfile [hunk 2]
```diff
@@ -321,80 +327,104 @@ pipeline {
...
+    }
+    stage("Send Slack Status") {
+      agent { label params['JENKINS_AGENT_LABEL'] }
+      when {
+        expression { params.SEND_SLACK == true }
+      }
+      steps {
+        script {
+          def slack_channel = "ocp-qe-scale-ci-results"
+          println "Sending Slack notification to #ocp-qe-scale-ci-results..."
```
Comment (nathan-weinberg) [inline Jenkinsfile:414]:
```suggestion
          println "Sending Slack notification to #${slack_channel}..."
```

**Comment by qiliRedHat** [comment]
@paigerube14 Pls past a test job link after you get a successful one.

**Comment by paigerube14** [comment]
@qiliRedHat passing job: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/kube-burner/152/

**Pull request merged**